core/src/main/java/hudson/model/Job.java

```diff
@@ -1070,6 +1070,60 @@ public PermalinkList getPermalinks() {
     }
 
     /**
+     * The following methods should be overriden by Jobs that support being disabled:
+     * <p><ul>
+     * <li>supportsDisable() - override to return true</li>
+     * <li>isDisabled()</li>
+     * <li>disable()</li>
+     * <li>enable()</li>
+     * </ul>
+     *
+     * <p>
+     * The default behavior for jobs that do not override these methods is isDisabled() always returns false,
```

Comment:
Smells wrong. I would expect this to be in `ParameterizedJobMixIn.ParameterizedJob` perhaps.